config/gdi-starter-kit/docker-compose.override.yml

```diff
@@ -25,7 +25,7 @@ services:
     healthcheck:
       test: curl -fq http://localhost:9000/minio/health/live
     restart: no
-    ports: !override []
+    # ports: !override []
```

Comment:
I would keep this or remap the ports to expose them here instead, ideally with an environment variable to be able to configure the external port. E.g., 
```
    ports: !override
      - ${PORT:-1234}:1234
```

Reply:
yes, sorry, this was just intended for testing, but happened to be commited. Reset in 01bd15d